saleor/graphql/order/types.py

```diff
@@ -696,7 +696,10 @@ class Order(ModelObjectType):
         TaxedMoney, description="Total price of shipping.", required=True
     )
     shipping_tax_rate = graphene.Float(required=True)
-    token = graphene.String(required=True)
+    token = graphene.String(
+        required=True,
+        deprecation_reason=(f"{DEPRECATED_IN_3X_FIELD} Use `id` instead."),
```

Comment:
As agreed on Slack let's leave this field without deprecation, it will be needed to get the token without decoding ID.

Reply:
We decided to leave it as it is.